[
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 1,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_works_in_mono_process_only_environment"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 3,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_invalid_config_return_code"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 4,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_beginning_backslash"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 6,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_async_as_identifier",
               "tests/test_black.py::BlackTestCase::test_python37"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 7,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_tuple_assign"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 8,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_comments7"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 10,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_comment_indentation"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 11,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_comments6"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 14,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_get_future_imports"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 15,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_fmtonoff2"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 18,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_preserves_line_endings"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 19,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_comment_in_decorator"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "black",
          "BUG_NUMBER": 23,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_black.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_black.py::BlackTestCase::test_python2"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "cookiecutter",
          "BUG_NUMBER": 1,
          "TARGET_DIR": "cookiecutter",
          "TEST_SUITE": [
               "tests/test_abort_generate_on_hook_error.py",
               "tests/test_generate_context.py",
               "tests/test_generate_copy_without_render.py",
               "tests/test_generate_file.py",
               "tests/test_generate_files.py",
               "tests/test_generate_hooks.py",
               "tests/test_hooks.py",
               "tests/test_output_folder.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_generate_context.py::test_generate_context_decodes_non_ascii_chars"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "cookiecutter",
          "BUG_NUMBER": 2,
          "TARGET_DIR": "cookiecutter",
          "TEST_SUITE": [
               "tests/test_find.py",
               "tests/test_hooks.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_hooks.py::TestFindHooks::test_find_hook",
               "tests/test_hooks.py::TestExternalHooks::test_run_hook"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "cookiecutter",
          "BUG_NUMBER": 3,
          "TARGET_DIR": "cookiecutter",
          "TEST_SUITE": [
               "tests/test_cli.py",
               "tests/test_prompt.py",
               "tests/test_read_repo_password.py",
               "tests/test_read_user_choice.py",
               "tests/test_read_user_dict.py",
               "tests/test_read_user_variable.py",
               "tests/test_read_user_yes_no.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_read_user_choice.py::test_click_invocation"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "cookiecutter",
          "BUG_NUMBER": 4,
          "TARGET_DIR": "cookiecutter",
          "TEST_SUITE": [
               "tests/test_find.py",
               "tests/test_generate_context.py",
               "tests/test_generate_copy_without_render.py",
               "tests/test_generate_file.py",
               "tests/test_generate_files.py",
               "tests/test_generate_hooks.py",
               "tests/test_hooks.py",
               "tests/test_output_folder.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_hooks.py::TestExternalHooks::test_run_failing_hook"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 1,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_additional_properties.py",
               "tests/test_additional_response_extra.py",
               "tests/test_additional_responses_bad.py",
               "tests/test_additional_responses_custom_validationerror.py",
               "tests/test_additional_responses_default_validationerror.py",
               "tests/test_additional_responses_response_class.py",
               "tests/test_additional_responses_router.py",
               "tests/test_callable_endpoint.py",
               "tests/test_custom_route_class.py",
               "tests/test_custom_swagger_ui_redirect.py",
               "tests/test_datetime_custom_encoder.py",
               "tests/test_default_response_class.py",
               "tests/test_default_response_class_router.py",
               "tests/test_dependency_cache.py",
               "tests/test_dependency_class.py",
               "tests/test_dependency_contextmanager.py",
               "tests/test_dependency_duplicates.py",
               "tests/test_dependency_overrides.py",
               "tests/test_duplicate_models_openapi.py",
               "tests/test_empty_router.py",
               "tests/test_extra_routes.py",
               "tests/test_filter_pydantic_sub_model.py",
               "tests/test_forms_from_non_typing_sequences.py",
               "tests/test_include_route.py",
               "tests/test_infer_param_optionality.py",
               "tests/test_inherited_custom_class.py",
               "tests/test_invalid_path_param.py",
               "tests/test_invalid_sequence_param.py",
               "tests/test_jsonable_encoder.py",
               "tests/test_multi_body_errors.py",
               "tests/test_multi_query_errors.py",
               "tests/test_no_swagger_ui_redirect.py",
               "tests/test_param_class.py",
               "tests/test_param_in_path_and_dependency.py",
               "tests/test_put_no_body.py",
               "tests/test_repeated_dependency_schema.py",
               "tests/test_request_body_parameters_media_type.py",
               "tests/test_response_change_status_code.py",
               "tests/test_response_class_no_mediatype.py",
               "tests/test_response_code_no_body.py",
               "tests/test_response_model_invalid.py",
               "tests/test_response_model_sub_types.py",
               "tests/test_router_events.py",
               "tests/test_router_prefix_with_template.py",
               "tests/test_security_api_key_cookie.py",
               "tests/test_security_api_key_cookie_optional.py",
               "tests/test_security_api_key_header.py",
               "tests/test_security_api_key_header_optional.py",
               "tests/test_security_api_key_query.py",
               "tests/test_security_api_key_query_optional.py",
               "tests/test_security_http_base.py",
               "tests/test_security_http_base_optional.py",
               "tests/test_security_http_basic_optional.py",
               "tests/test_security_http_basic_realm.py",
               "tests/test_security_http_bearer.py",
               "tests/test_security_http_bearer_optional.py",
               "tests/test_security_http_digest.py",
               "tests/test_security_http_digest_optional.py",
               "tests/test_security_oauth2.py",
               "tests/test_security_oauth2_authorization_code_bearer.py",
               "tests/test_security_oauth2_optional.py",
               "tests/test_security_oauth2_password_bearer_optional.py",
               "tests/test_security_openid_connect.py",
               "tests/test_security_openid_connect_optional.py",
               "tests/test_serialize_response.py",
               "tests/test_serialize_response_dataclass.py",
               "tests/test_serialize_response_model.py",
               "tests/test_skip_defaults.py",
               "tests/test_starlette_exception.py",
               "tests/test_starlette_urlconvertors.py",
               "tests/test_sub_callbacks.py",
               "tests/test_swagger_ui_init_oauth.py",
               "tests/test_union_body.py",
               "tests/test_union_inherited_body.py",
               "tests/test_validate_response.py",
               "tests/test_validate_response_dataclass.py",
               "tests/test_validate_response_recursive.py",
               "tests/test_ws_router.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_jsonable_encoder.py::test_encode_model_with_default"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 2,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_additional_response_extra.py",
               "tests/test_additional_responses_router.py",
               "tests/test_custom_route_class.py",
               "tests/test_default_response_class.py",
               "tests/test_default_response_class_router.py",
               "tests/test_dependency_overrides.py",
               "tests/test_empty_router.py",
               "tests/test_include_route.py",
               "tests/test_infer_param_optionality.py",
               "tests/test_router_events.py",
               "tests/test_router_prefix_with_template.py",
               "tests/test_sub_callbacks.py",
               "tests/test_ws_router.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_ws_router.py::test_router_ws_depends_with_override"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 3,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_serialize_response_model.py",
               "tests/test_additional_response_extra.py",
               "tests/test_additional_responses_router.py",
               "tests/test_custom_route_class.py",
               "tests/test_default_response_class.py",
               "tests/test_default_response_class_router.py",
               "tests/test_dependency_overrides.py",
               "tests/test_empty_router.py",
               "tests/test_include_route.py",
               "tests/test_infer_param_optionality.py",
               "tests/test_router_events.py",
               "tests/test_router_prefix_with_template.py",
               "tests/test_sub_callbacks.py",
               "tests/test_ws_router.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_serialize_response_model.py::test_valid",
               "tests/test_serialize_response_model.py::test_coerce",
               "tests/test_serialize_response_model.py::test_validlist",
               "tests/test_serialize_response_model.py::test_validdict",
               "tests/test_serialize_response_model.py::test_valid_exclude_unset",
               "tests/test_serialize_response_model.py::test_coerce_exclude_unset",
               "tests/test_serialize_response_model.py::test_validlist_exclude_unset",
               "tests/test_serialize_response_model.py::test_validdict_exclude_unset"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 4,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_param_in_path_and_dependency.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_param_in_path_and_dependency.py::test_reused_param"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 5,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_filter_pydantic_sub_model.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_filter_pydantic_sub_model.py::test_filter_sub_model"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 6,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_forms_from_non_typing_sequences.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_forms_from_non_typing_sequences.py::test_python_list_param_as_form",
               "tests/test_forms_from_non_typing_sequences.py::test_python_set_param_as_form",
               "tests/test_forms_from_non_typing_sequences.py::test_python_tuple_param_as_form"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 7,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_multi_body_errors.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_multi_body_errors.py::test_jsonable_encoder_requiring_error"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 9,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_request_body_parameters_media_type.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_request_body_parameters_media_type.py::test_openapi_schema"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 10,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_skip_defaults.py",
               "tests/test_additional_response_extra.py",
               "tests/test_additional_responses_router.py",
               "tests/test_dependency_overrides.py",
               "tests/test_empty_router.py",
               "tests/test_include_route.py",
               "tests/test_router_prefix_with_template.py",
               "tests/test_ws_router.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_skip_defaults.py::test_return_defaults"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 11,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_union_inherited_body.py",
               "tests/test_union_body.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_union_body.py::test_item_openapi_schema",
               "tests/test_union_body.py::test_post_other_item",
               "tests/test_union_body.py::test_post_item",
               "tests/test_union_inherited_body.py::test_inherited_item_openapi_schema",
               "tests/test_union_inherited_body.py::test_post_extended_item",
               "tests/test_union_inherited_body.py::test_post_item"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 12,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_security_http_base.py",
               "tests/test_security_http_base_optional.py",
               "tests/test_security_http_basic_optional.py",
               "tests/test_security_http_basic_realm.py",
               "tests/test_security_http_bearer.py",
               "tests/test_security_http_bearer_optional.py",
               "tests/test_security_http_digest.py",
               "tests/test_security_http_digest_optional.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_security_http_bearer_optional.py::test_security_http_bearer_incorrect_scheme_credentials"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 14,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_additional_properties.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_additional_properties.py::test_additional_properties_schema"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "fastapi",
          "BUG_NUMBER": 16,
          "TARGET_DIR": "fastapi",
          "TEST_SUITE": [
               "tests/test_jsonable_encoder.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_jsonable_encoder.py::test_encode_model_with_config"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "httpie",
          "BUG_NUMBER": 2,
          "TARGET_DIR": "httpie",
          "TEST_SUITE": [
               "tests/test_redirects.py",
               "tests/test_errors.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_redirects.py::TestRedirects::test_max_redirects"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "httpie",
          "BUG_NUMBER": 3,
          "TARGET_DIR": "httpie",
          "TEST_SUITE": [
               "tests/test_sessions.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_sessions.py::TestSession::test_download_in_session"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "httpie",
          "BUG_NUMBER": 4,
          "TARGET_DIR": "httpie",
          "TEST_SUITE": [
               "tests/test_regressions.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_regressions.py::test_Host_header_overwrite"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "httpie",
          "BUG_NUMBER": 5,
          "TARGET_DIR": "httpie",
          "TEST_SUITE": [
               "tests/tests.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/tests.py::TestItemParsing::test_escape_longsep"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 1,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/backend/backend_test.py",
               "tests/keras/callbacks/callbacks_test.py",
               "tests/keras/engine/test_topology.py",
               "tests/keras/initializers_test.py",
               "tests/keras/layers/cudnn_recurrent_test.py",
               "tests/keras/layers/merge_test.py",
               "tests/keras/layers/recurrent_test.py",
               "tests/test_model_saving.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/initializers_test.py::test_statefulness"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 3,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/integration_tests/test_tensorflow_integration.py",
               "tests/keras/layers/cudnn_recurrent_test.py",
               "tests/keras/layers/merge_test.py",
               "tests/keras/layers/recurrent_test.py",
               "tests/keras/losses_test.py",
               "tests/keras/test_sequential_model.py",
               "tests/keras/utils/multi_gpu_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/test_sequential_model.py::test_clone_functional_model_with_multi_outputs"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 5,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/merge_test.py",
               "tests/keras/utils/data_utils_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/utils/data_utils_test.py::test_data_utils"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 6,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/engine/test_training.py",
               "tests/keras/layers/merge_test.py",
               "tests/keras/test_sequential_model.py",
               "tests/test_loss_masking.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_loss_masking.py::test_masking_is_all_zeros"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 7,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/merge_test.py",
               "tests/keras/wrappers/scikit_learn_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/wrappers/scikit_learn_test.py::test_regression_predict_shape_correct_num_test_1"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 10,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/engine/test_training.py",
               "tests/keras/layers/merge_test.py",
               "tests/keras/test_sequential_model.py",
               "tests/test_loss_masking.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/engine/test_training.py::test_sample_weights"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 14,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/merge_test.py",
               "tests/keras/metrics_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/metrics_test.py::test_sparse_top_k_categorical_accuracy"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 17,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/merge_test.py",
               "tests/keras/metrics_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/metrics_test.py::test_sparse_categorical_accuracy_correctness"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 18,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/backend/backend_test.py",
               "tests/keras/layers/merge_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/backend/backend_test.py::TestBackend::test_function_tf_run_options_with_run_metadata"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 20,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/backend/backend_test.py",
               "tests/keras/engine/test_topology.py",
               "tests/keras/engine/test_training.py",
               "tests/keras/layers/convolutional_recurrent_test.py",
               "tests/keras/layers/convolutional_test.py",
               "tests/keras/layers/merge_test.py",
               "tests/keras/layers/recurrent_test.py",
               "tests/keras/legacy/interface_test.py",
               "tests/keras/test_callbacks.py",
               "tests/keras/utils/conv_utils_test.py",
               "tests/keras/utils/layer_utils_test.py",
               "tests/keras/utils/vis_utils_test.py",
               "tests/test_model_saving.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/layers/convolutional_test.py::test_conv2d_transpose_dilation"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 26,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/backend/backend_test.py",
               "tests/keras/layers/merge_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/backend/backend_test.py::TestBackend::test_rnn_additional_states"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 31,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/backend/backend_test.py",
               "tests/keras/layers/merge_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/backend/backend_test.py::TestBackend::test_ctc"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 33,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/merge_test.py",
               "tests/keras/preprocessing/text_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/preprocessing/text_test.py::test_text_to_word_sequence_multichar_split",
               "tests/keras/preprocessing/text_test.py::test_text_to_word_sequence_unicode_multichar_split"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 35,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/merge_test.py",
               "tests/keras/preprocessing/image_test.py",
               "tests/keras/utils/multi_gpu_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/preprocessing/image_test.py::TestImage::test_directory_iterator"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 38,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/cudnn_recurrent_test.py",
               "tests/keras/layers/merge_test.py",
               "tests/keras/layers/recurrent_test.py",
               "tests/keras/layers/wrappers_test.py",
               "tests/keras/legacy/interface_test.py",
               "tests/keras/legacy/layers_test.py",
               "tests/keras/utils/vis_utils_test.py",
               "tests/test_loss_weighting.py",
               "tests/test_model_saving.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/layers/recurrent_test.py::test_minimal_rnn_cell_layer"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 40,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/cudnn_recurrent_test.py",
               "tests/keras/layers/merge_test.py",
               "tests/keras/layers/recurrent_test.py",
               "tests/keras/layers/wrappers_test.py",
               "tests/keras/legacy/interface_test.py",
               "tests/keras/legacy/layers_test.py",
               "tests/keras/utils/vis_utils_test.py",
               "tests/test_loss_weighting.py",
               "tests/test_model_saving.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/layers/recurrent_test.py::test_stacked_rnn_compute_output_shape"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 41,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/keras/layers/merge_test.py",
               "tests/keras/utils/data_utils_test.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/utils/data_utils_test.py::test_generator_enqueuer_fail_threads"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "keras",
          "BUG_NUMBER": 43,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/integration_tests/test_image_data_tasks.py",
               "tests/integration_tests/test_temporal_data_tasks.py",
               "tests/integration_tests/test_vector_data_tasks.py",
               "tests/keras/layers/merge_test.py",
               "tests/keras/legacy/models_test.py",
               "tests/keras/optimizers_test.py",
               "tests/keras/regularizers_test.py",
               "tests/keras/test_callbacks.py",
               "tests/keras/test_sequential_model.py",
               "tests/keras/utils/np_utils_test.py",
               "tests/test_loss_weighting.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/keras/utils/np_utils_test.py::test_to_categorical"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 2,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/indexing/test_scalar.py",
               "pandas/tests/indexing/multiindex/test_slice.py",
               "pandas/tests/indexing/test_indexing.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/indexing/test_scalar.py::test_at_with_tuple_index_get",
               "pandas/tests/indexing/test_scalar.py::test_at_with_tuple_index_set",
               "pandas/tests/indexing/test_scalar.py::test_multiindex_at_get",
               "pandas/tests/indexing/test_scalar.py::test_multiindex_at_set"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 12,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/frame/methods/test_cov_corr.py",
               "pandas/tests/frame/test_validate.py",
               "pandas/tests/io/test_stata.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/frame/methods/test_cov_corr.py::TestDataFrameCov::test_cov_nullable_integer"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 23,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/indexes/datetimes/test_setops.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/indexes/datetimes/test_setops.py::TestDatetimeIndexSetOps::test_intersection_empty",
               "pandas/tests/indexes/datetimes/test_setops.py::TestBusinessDatetimeIndex::test_intersection_bug",
               "pandas/tests/indexes/datetimes/test_setops.py::TestCustomDatetimeIndex::test_intersection_bug"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 43,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/frame/test_arithmetic.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/frame/test_arithmetic.py::test_pow_with_realignment"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 54,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/arrays/categorical/test_constructors.py",
               "pandas/tests/arrays/categorical/test_dtypes.py",
               "pandas/tests/arrays/categorical/test_missing.py",
               "pandas/tests/arrays/test_array.py",
               "pandas/tests/arrays/test_datetimes.py",
               "pandas/tests/arrays/test_period.py",
               "pandas/tests/dtypes/cast/test_construct_from_scalar.py",
               "pandas/tests/dtypes/cast/test_find_common_type.py",
               "pandas/tests/dtypes/cast/test_promote.py",
               "pandas/tests/dtypes/test_common.py",
               "pandas/tests/dtypes/test_dtypes.py",
               "pandas/tests/dtypes/test_missing.py",
               "pandas/tests/extension/test_common.py",
               "pandas/tests/extension/test_datetime.py",
               "pandas/tests/extension/test_interval.py",
               "pandas/tests/extension/test_period.py",
               "pandas/tests/frame/indexing/test_categorical.py",
               "pandas/tests/frame/test_dtypes.py",
               "pandas/tests/frame/test_timezones.py",
               "pandas/tests/indexes/categorical/test_category.py",
               "pandas/tests/indexes/interval/test_astype.py",
               "pandas/tests/indexes/interval/test_constructors.py",
               "pandas/tests/indexes/multi/test_astype.py",
               "pandas/tests/indexes/period/test_constructors.py",
               "pandas/tests/indexing/test_categorical.py",
               "pandas/tests/io/json/test_json_table_schema.py",
               "pandas/tests/io/parser/test_dtypes.py",
               "pandas/tests/reshape/merge/test_merge.py",
               "pandas/tests/reshape/test_concat.py",
               "pandas/tests/series/test_constructors.py",
               "pandas/tests/series/test_dtypes.py",
               "pandas/tests/test_algos.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/dtypes/test_dtypes.py::TestCategoricalDtype::test_from_values_or_dtype_invalid_dtype"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 56,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/indexing/test_scalar.py",
               "pandas/tests/frame/test_validate.py",
               "pandas/tests/io/test_stata.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/indexing/test_scalar.py::test_iat_dont_wrap_object_datetimelike"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 68,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/arrays/interval/test_interval.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/arrays/interval/test_interval.py::TestMethods::test_shift"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 72,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/frame/indexing/test_categorical.py",
               "pandas/tests/extension/test_external_block.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/frame/indexing/test_categorical.py::TestDataFrameIndexingCategorical::test_setitem_single_row_categorical"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 74,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/indexes/timedeltas/test_constructors.py",
               "pandas/tests/plotting/test_datetimelike.py",
               "pandas/tests/resample/test_timedelta.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/indexes/timedeltas/test_constructors.py::TestTimedeltaIndex::test_infer_from_tdi_mismatch"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 77,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/arithmetic/test_array_ops.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/arithmetic/test_array_ops.py::test_na_logical_op_2d"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 81,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/arrays/test_integer.py",
               "pandas/tests/extension/test_integer.py",
               "pandas/tests/frame/test_block_internals.py",
               "pandas/tests/frame/test_dtypes.py",
               "pandas/tests/frame/test_reshape.py",
               "pandas/tests/indexing/test_indexing.py",
               "pandas/tests/plotting/test_frame.py",
               "pandas/tests/reshape/test_concat.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/arrays/test_integer.py::TestCasting::test_astype_boolean"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 84,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/frame/test_reshape.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/frame/test_reshape.py::TestDataFrameReshape::test_unstack_tuplename_in_multiindex",
               "pandas/tests/frame/test_reshape.py::TestDataFrameReshape::test_unstack_mixed_type_name_in_multiindex"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 116,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/reshape/merge/test_merge_asof.py",
               "pandas/tests/reshape/merge/test_merge.py",
               "pandas/tests/reshape/merge/test_multi.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/reshape/merge/test_merge_asof.py::TestAsOfMerge::test_merge_index_column_tz"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 137,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/reshape/merge/test_merge.py",
               "pandas/tests/extension/test_categorical.py",
               "pandas/tests/arrays/categorical/test_api.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/extension/test_categorical.py::TestCasting::test_cast_category_to_extension_dtype",
               "pandas/tests/reshape/merge/test_merge.py::test_merge_on_cat_and_ext_array"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 141,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/indexes/test_range.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/indexes/test_range.py::TestRangeIndex::test_get_indexer_decreasing"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 143,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/indexes/test_range.py",
               "pandas/tests/frame/test_indexing.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/frame/test_indexing.py::TestDataFrameIndexing::test_reindex_limit",
               "pandas/tests/indexes/test_range.py::TestRangeIndex::test_get_indexer_limit"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 146,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/dtypes/test_missing.py",
               "pandas/tests/generic/test_label_or_level_utils.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/dtypes/test_missing.py::test_array_equivalent_tzawareness"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "pandas",
          "BUG_NUMBER": 168,
          "TARGET_DIR": "pandas",
          "TEST_SUITE": [
               "pandas/tests/groupby/test_groupby.py",
               "pandas/tests/groupby/aggregate/test_aggregate.py",
               "pandas/tests/groupby/test_grouping.py",
               "pandas/tests/groupby/test_timegrouper.py",
               "pandas/tests/resample/test_base.py",
               "pandas/tests/resample/test_datetime_index.py",
               "pandas/tests/resample/test_time_grouper.py"
          ],
          "EXCLUDE": [
               "pandas/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "pandas/tests/groupby/test_groupby.py::test_groupby_axis_1"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "sanic",
          "BUG_NUMBER": 1,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_blueprints.py",
               "tests/test_app.py",
               "tests/test_asgi.py",
               "tests/test_config.py",
               "tests/test_custom_request.py",
               "tests/test_exceptions.py",
               "tests/test_exceptions_handler.py",
               "tests/test_keep_alive_timeout.py",
               "tests/test_logging.py",
               "tests/test_request_timeout.py",
               "tests/test_requests.py",
               "tests/test_response.py",
               "tests/test_response_timeout.py",
               "tests/test_routes.py",
               "tests/test_worker.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_blueprints.py::test_bp_middleware_order"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "sanic",
          "BUG_NUMBER": 3,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_url_for.py",
               "tests/test_asgi.py",
               "tests/test_config.py",
               "tests/test_custom_request.py",
               "tests/test_exceptions.py",
               "tests/test_exceptions_handler.py",
               "tests/test_keep_alive_timeout.py",
               "tests/test_logging.py",
               "tests/test_request_timeout.py",
               "tests/test_requests.py",
               "tests/test_response.py",
               "tests/test_response_timeout.py",
               "tests/test_routes.py",
               "tests/test_worker.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_url_for.py::test_routes_with_host"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "sanic",
          "BUG_NUMBER": 4,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "tests/test_requests.py"
          ],
          "EXCLUDE": [
               "env",
               "tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tests/test_requests.py::test_url_for_without_server_name"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "spacy",
          "BUG_NUMBER": 1,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "spacy/tests/test_errors.py",
               "spacy/tests/test_displacy.py",
               "spacy/tests/test_gold.py"
          ],
          "EXCLUDE": [
               "spacy/tests",
               "env"
          ],
          "TARGET_FAILING_TESTS": [
               "spacy/tests/test_errors.py::test_add_codes"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "spacy",
          "BUG_NUMBER": 2,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "spacy/tests/regression/test_issue5137.py",
               "spacy/tests/doc/test_creation.py",
               "spacy/tests/doc/test_span.py",
               "spacy/tests/doc/test_to_json.py",
               "spacy/tests/lang/en/test_customized_tokenizer.py",
               "spacy/tests/lang/en/test_punct.py",
               "spacy/tests/lang/test_initialize.py",
               "spacy/tests/matcher/test_pattern_validation.py",
               "spacy/tests/parser/test_add_label.py",
               "spacy/tests/regression/test_issue3001-3500.py",
               "spacy/tests/regression/test_issue3611.py",
               "spacy/tests/regression/test_issue4030.py",
               "spacy/tests/regression/test_issue4042.py",
               "spacy/tests/regression/test_issue4054.py",
               "spacy/tests/regression/test_issue4190.py",
               "spacy/tests/regression/test_issue4348.py",
               "spacy/tests/regression/test_issue4674.py",
               "spacy/tests/regression/test_issue4707.py",
               "spacy/tests/serialize/test_serialize_kb.py",
               "spacy/tests/serialize/test_serialize_tokenizer.py",
               "spacy/tests/test_gold.py",
               "spacy/tests/test_json_schemas.py",
               "spacy/tests/test_misc.py",
               "spacy/tests/tokenizer/test_explain.py",
               "spacy/tests/tokenizer/test_tokenizer.py"
          ],
          "EXCLUDE": [
               "spacy/tests",
               "env"
          ],
          "TARGET_FAILING_TESTS": [
               "spacy/tests/regression/test_issue5137.py::test_issue5137"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "spacy",
          "BUG_NUMBER": 3,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "spacy/tests/regression/test_issue5314.py",
               "spacy/tests/test_gold.py"
          ],
          "EXCLUDE": [
               "spacy/tests",
               "env"
          ],
          "TARGET_FAILING_TESTS": [
               "spacy/tests/regression/test_issue5314.py::test_issue5314"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "spacy",
          "BUG_NUMBER": 4,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "spacy/tests/regression/test_issue4665.py",
               "spacy/tests/test_cli.py",
               "spacy/tests/test_gold.py"
          ],
          "EXCLUDE": [
               "spacy/tests",
               "env"
          ],
          "TARGET_FAILING_TESTS": [
               "spacy/tests/regression/test_issue4665.py::test_issue4665"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "spacy",
          "BUG_NUMBER": 5,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "spacy/tests/doc/test_pickle_doc.py",
               "spacy/tests/lang/fr/test_prefix_suffix_infix.py",
               "spacy/tests/parser/test_nn_beam.py",
               "spacy/tests/pipeline/test_analysis.py",
               "spacy/tests/pipeline/test_entity_ruler.py",
               "spacy/tests/pipeline/test_factories.py",
               "spacy/tests/pipeline/test_pipe_methods.py",
               "spacy/tests/pipeline/test_textcat.py",
               "spacy/tests/regression/test_issue1-1000.py",
               "spacy/tests/regression/test_issue1501-2000.py",
               "spacy/tests/regression/test_issue2501-3000.py",
               "spacy/tests/regression/test_issue3526.py",
               "spacy/tests/regression/test_issue4278.py",
               "spacy/tests/serialize/test_serialize_language.py",
               "spacy/tests/test_gold.py",
               "spacy/tests/test_language.py",
               "spacy/tests/test_lemmatizer.py"
          ],
          "EXCLUDE": [
               "spacy/tests",
               "env"
          ],
          "TARGET_FAILING_TESTS": [
               "spacy/tests/test_language.py::test_evaluate_no_pipe"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "spacy",
          "BUG_NUMBER": 6,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "spacy/tests/doc/test_pickle_doc.py",
               "spacy/tests/lang/fr/test_prefix_suffix_infix.py",
               "spacy/tests/parser/test_nn_beam.py",
               "spacy/tests/pipeline/test_analysis.py",
               "spacy/tests/pipeline/test_entity_ruler.py",
               "spacy/tests/pipeline/test_factories.py",
               "spacy/tests/pipeline/test_pipe_methods.py",
               "spacy/tests/pipeline/test_textcat.py",
               "spacy/tests/regression/test_issue1-1000.py",
               "spacy/tests/regression/test_issue1501-2000.py",
               "spacy/tests/regression/test_issue2501-3000.py",
               "spacy/tests/regression/test_issue3526.py",
               "spacy/tests/regression/test_issue4278.py",
               "spacy/tests/serialize/test_serialize_language.py",
               "spacy/tests/test_gold.py",
               "spacy/tests/test_language.py",
               "spacy/tests/test_lemmatizer.py"
          ],
          "EXCLUDE": [
               "spacy/tests",
               "env"
          ],
          "TARGET_FAILING_TESTS": [
               "spacy/tests/pipeline/test_analysis.py::test_analysis_validate_attrs_remove_pipe"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 2,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_utils.py::test_get_all_executables_pathsep"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 3,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/shells/test_fish.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/shells/test_fish.py::TestFish::test_info"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 4,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/shells/test_fish.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/shells/test_fish.py::TestFish::test_get_aliases"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 6,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_git_branch_exists.py",
               "tests/specific/test_git.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_git_branch_exists.py::test_get_new_command"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 7,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_php_s.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_php_s.py::test_match"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 8,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_dnf_no_such_command.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_dnf_no_such_command.py::test_get_new_command",
               "tests/rules/test_dnf_no_such_command.py::test_get_operations"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 13,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_git_branch_exists.py",
               "tests/specific/test_git.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_git_branch_exists.py::test_match",
               "tests/rules/test_git_branch_exists.py::test_get_new_command"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 14,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/shells/test_fish.py",
               "tests/rules/test_git_branch_list.py",
               "tests/specific/test_git.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/shells/test_fish.py::TestFish::test_get_overridden_aliases"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 15,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_git_add.py",
               "tests/specific/test_git.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_git_add.py::test_match"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 17,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/shells/test_bash.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/shells/test_bash.py::TestBash::test_get_aliases",
               "tests/shells/test_bash.py::TestBash::test_from_shell"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 19,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_git_push.py",
               "tests/rules/test_git_push_force.py",
               "tests/specific/test_git.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_git_push_force.py::test_get_new_command"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 20,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_dirty_unzip.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_dirty_unzip.py::test_get_new_command"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 25,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_mkdir_p.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_mkdir_p.py::test_get_new_command"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 26,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_vagrant_up.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_vagrant_up.py::test_get_new_command"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 28,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/rules/test_fix_file.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/rules/test_fix_file.py::test_get_new_command_with_settings"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "thefuck",
          "BUG_NUMBER": 29,
          "TARGET_DIR": "thefuck",
          "TEST_SUITE": [
               "tests/test_utils.py",
               "tests/test_types.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "tests/test_types.py::test_update_settings",
               "tests/test_utils.py::test_wrap_settings"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "tornado",
          "BUG_NUMBER": 1,
          "TARGET_DIR": "tornado",
          "TEST_SUITE": [
               "tornado/test/websocket_test.py"
          ],
          "EXCLUDE": [
               "tornado/test"
          ],
          "TARGET_FAILING_TESTS": [
               "tornado/test/websocket_test.py::WebSocketTest::test_nodelay"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "tornado",
          "BUG_NUMBER": 5,
          "TARGET_DIR": "tornado",
          "TEST_SUITE": [
               "tornado/test/asyncio_test.py",
               "tornado/test/concurrent_test.py",
               "tornado/test/gen_test.py",
               "tornado/test/http1connection_test.py",
               "tornado/test/httpclient_test.py",
               "tornado/test/httpserver_test.py",
               "tornado/test/ioloop_test.py",
               "tornado/test/iostream_test.py",
               "tornado/test/locks_test.py",
               "tornado/test/process_test.py",
               "tornado/test/queues_test.py",
               "tornado/test/runtests.py",
               "tornado/test/simple_httpclient_test.py",
               "tornado/test/stack_context_test.py",
               "tornado/test/tcpclient_test.py",
               "tornado/test/testing_test.py",
               "tornado/test/twisted_test.py",
               "tornado/test/util_test.py",
               "tornado/test/web_test.py"
          ],
          "EXCLUDE": [
               "tornado/test"
          ],
          "TARGET_FAILING_TESTS": [
               "tornado/test/ioloop_test.py::TestPeriodicCallbackMath::test_clock_backwards"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "tornado",
          "BUG_NUMBER": 6,
          "TARGET_DIR": "tornado",
          "TEST_SUITE": [
               "tornado/test/asyncio_test.py",
               "tornado/test/concurrent_test.py",
               "tornado/test/gen_test.py",
               "tornado/test/http1connection_test.py",
               "tornado/test/httpclient_test.py",
               "tornado/test/httpserver_test.py",
               "tornado/test/ioloop_test.py",
               "tornado/test/iostream_test.py",
               "tornado/test/locks_test.py",
               "tornado/test/process_test.py",
               "tornado/test/queues_test.py",
               "tornado/test/runtests.py",
               "tornado/test/simple_httpclient_test.py",
               "tornado/test/stack_context_test.py",
               "tornado/test/tcpclient_test.py",
               "tornado/test/testing_test.py",
               "tornado/test/twisted_test.py",
               "tornado/test/util_test.py",
               "tornado/test/web_test.py"
          ],
          "EXCLUDE": [
               "tornado/test"
          ],
          "TARGET_FAILING_TESTS": [
               "tornado/test/asyncio_test.py::LeakTest::test_ioloop_close_leak",
               "tornado/test/asyncio_test.py::LeakTest::test_asyncio_close_leak"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "tornado",
          "BUG_NUMBER": 8,
          "TARGET_DIR": "tornado",
          "TEST_SUITE": [
               "tornado/test/websocket_test.py"
          ],
          "EXCLUDE": [
               "tornado/test"
          ],
          "TARGET_FAILING_TESTS": [
               "tornado/test/websocket_test.py::WebSocketTest::test_missing_websocket_key"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "tqdm",
          "BUG_NUMBER": 3,
          "TARGET_DIR": "tqdm",
          "TEST_SUITE": [
               "tqdm/tests/tests_pandas.py",
               "tqdm/tests/tests_perf.py",
               "tqdm/tests/tests_synchronisation.py",
               "tqdm/tests/tests_tqdm.py"
          ],
          "EXCLUDE": [
               "tqdm/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tqdm/tests/tests_tqdm.py::test_bool"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "tqdm",
          "BUG_NUMBER": 4,
          "TARGET_DIR": "tqdm",
          "TEST_SUITE": [
               "tqdm/tests/tests_pandas.py",
               "tqdm/tests/tests_perf.py",
               "tqdm/tests/tests_synchronisation.py",
               "tqdm/tests/tests_tqdm.py"
          ],
          "EXCLUDE": [
               "tqdm/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tqdm/tests/tests_tqdm.py::test_nototal"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "tqdm",
          "BUG_NUMBER": 5,
          "TARGET_DIR": "tqdm",
          "TEST_SUITE": [
               "tqdm/tests/tests_pandas.py",
               "tqdm/tests/tests_perf.py",
               "tqdm/tests/tests_synchronisation.py",
               "tqdm/tests/tests_tqdm.py"
          ],
          "EXCLUDE": [
               "tqdm/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tqdm/tests/tests_tqdm.py::test_bool"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "tqdm",
          "BUG_NUMBER": 6,
          "TARGET_DIR": "tqdm",
          "TEST_SUITE": [
               "tqdm/tests/tests_pandas.py",
               "tqdm/tests/tests_perf.py",
               "tqdm/tests/tests_synchronisation.py",
               "tqdm/tests/tests_tqdm.py"
          ],
          "EXCLUDE": [
               "tqdm/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tqdm/tests/tests_synchronisation.py::test_imap"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "tqdm",
          "BUG_NUMBER": 7,
          "TARGET_DIR": "tqdm",
          "TEST_SUITE": [
               "tqdm/tests/tests_main.py"
          ],
          "EXCLUDE": [
               "tqdm/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tqdm/tests/tests_main.py::test_main"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "tqdm",
          "BUG_NUMBER": 8,
          "TARGET_DIR": "tqdm",
          "TEST_SUITE": [
               "tqdm/tests/tests_pandas.py",
               "tqdm/tests/tests_perf.py",
               "tqdm/tests/tests_tqdm.py"
          ],
          "EXCLUDE": [
               "tqdm/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tqdm/tests/tests_tqdm.py::test_format_meter"
          ]
     },
     {
          "PYTHON_V": 3.6,
          "BENCHMARK_NAME": "tqdm",
          "BUG_NUMBER": 9,
          "TARGET_DIR": "tqdm",
          "TEST_SUITE": [
               "tqdm/tests/tests_tqdm.py"
          ],
          "EXCLUDE": [
               "tqdm/tests"
          ],
          "TARGET_FAILING_TESTS": [
               "tqdm/tests/tests_tqdm.py::test_si_format",
               "tqdm/tests/tests_tqdm.py::test_update"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 1,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_downloader_http.py",
               "test/test_execution.py",
               "test/test_update.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_match_str"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 2,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_all_urls.py",
               "test/test_iqiyi_sdk_interpreter.py",
               "test/test_youtube_lists.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_InfoExtractor.py::TestInfoExtractor::test_parse_mpd_formats"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 6,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_compat.py",
               "test/test_execution.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_parse_dfxp_time_expr"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 7,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_compat.py",
               "test/test_execution.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_js_to_json_realworld"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 13,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_YoutubeDLCookieJar.py",
               "test/test_aes.py",
               "test/test_downloader_http.py",
               "test/test_execution.py",
               "test/test_update.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_urljoin"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 15,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_execution.py",
               "test/test_update.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_js_to_json_edgecases",
               "test/test_utils.py::TestUtil::test_js_to_json_realworld"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 17,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_execution.py",
               "test/test_update.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_cli_bool_option"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 18,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_YoutubeDL.py",
               "test/test_age_restriction.py",
               "test/test_http.py",
               "test/test_postprocessors.py",
               "test/test_socks.py",
               "test/test_subtitles.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_YoutubeDL.py::TestYoutubeDL::test_do_not_override_ie_key_in_url_transparent"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 20,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_execution.py",
               "test/test_update.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_get_element_by_attribute"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 21,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_execution.py",
               "test/test_update.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_urljoin"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 24,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_execution.py",
               "test/test_update.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_YoutubeDL.py::TestYoutubeDL::test_match_filter"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 25,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_InfoExtractor.py",
               "test/test_YoutubeDL.py",
               "test/test_aes.py",
               "test/test_execution.py",
               "test/test_update.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_js_to_json_realworld"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 31,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_compat.py",
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_parse_duration"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 34,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_js_to_json"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 41,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_unified_dates"
          ]
     },
     {
          "PYTHON_V": 3.7,
          "BENCHMARK_NAME": "youtube-dl",
          "BUG_NUMBER": 43,
          "TARGET_DIR": "youtube_dl",
          "TEST_SUITE": [
               "test/test_utils.py"
          ],
          "EXCLUDE": [],
          "TARGET_FAILING_TESTS": [
               "test/test_utils.py::TestUtil::test_url_basename"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 3,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/choice_parameter_test.py",
               "test/contrib/streaming_test.py",
               "test/date_interval_test.py",
               "test/date_parameter_test.py",
               "test/dict_parameter_test.py",
               "test/numerical_parameter_test.py",
               "test/parameter_test.py",
               "test/task_bulk_complete_test.py",
               "test/worker_multiprocess_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/parameter_test.py::TestSerializeTupleParameter::testSerialize"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 5,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/decorator_test.py",
               "test/util_previous_test.py",
               "test/util_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/util_test.py::BasicsTest::test_inherits_has_effect_MRO",
               "test/util_test.py::BasicsTest::test_requires_has_effect_MRO"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 6,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/choice_parameter_test.py",
               "test/contrib/streaming_test.py",
               "test/date_interval_test.py",
               "test/date_parameter_test.py",
               "test/dict_parameter_test.py",
               "test/numerical_parameter_test.py",
               "test/parameter_test.py",
               "test/task_bulk_complete_test.py",
               "test/worker_multiprocess_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/parameter_test.py::TestParametersHashability::test_list_dict",
               "test/parameter_test.py::TestParametersHashability::test_tuple_dict"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 12,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/hdfs_client_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/hdfs_client_test.py::HdfsClientTest::test_get_autoconfig_client_cached"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 13,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/cmdline_test.py",
               "test/contrib/hdfs_test.py",
               "test/contrib/hive_test.py",
               "test/contrib/opener_test.py",
               "test/contrib/test_ssh.py",
               "test/file_test.py",
               "test/scheduler_visualisation_test.py",
               "test/task_test.py",
               "test/test_ssh.py",
               "test/worker_external_task_test.py",
               "test/wrap_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/file_test.py::FileSystemTest::test_move_to_new_dir"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 22,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/scheduler_test.py",
               "test/central_planner_test.py",
               "test/rpc_test.py",
               "test/scheduler_visualisation_test.py",
               "test/server_test.py",
               "test/task_history_test.py",
               "test/task_test.py",
               "test/worker_external_task_test.py",
               "test/worker_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/scheduler_test.py::SchedulerTest::test_worker_prune_after_init"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 23,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/central_planner_test.py",
               "test/cmdline_test.py",
               "test/date_hour_test.py",
               "test/date_interval_test.py",
               "test/date_minute_test.py",
               "test/dynamic_import_test.py",
               "test/event_callbacks_test.py",
               "test/import_test.py",
               "test/interface_test.py",
               "test/rpc_test.py",
               "test/scheduler_visualisation_test.py",
               "test/server_test.py",
               "test/task_history_test.py",
               "test/task_test.py",
               "test/worker_external_task_test.py",
               "test/worker_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/worker_external_task_test.py::WorkerExternalTaskTest::test_external_dependency_completes_later"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 26,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/contrib/hadoop_jar_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/contrib/hadoop_jar_test.py::HadoopJarJobTaskTest::test_missing_jar"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 28,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/contrib/hive_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/contrib/hive_test.py::HiveCommandClientTest::test_table_exists",
               "test/contrib/hive_test.py::HiveCommandClientTest::test_apacheclient_table_exists"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 29,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/decorator_test.py",
               "test/task_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/task_test.py::TaskTest::test_external_tasks_loadable"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 30,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/test_event_callbacks.py",
               "test/fib_test.py",
               "test/instance_test.py",
               "test/instance_wrap_test.py",
               "test/interface_test.py",
               "test/recursion_test.py",
               "test/scheduler_visualisation_test.py",
               "test/task_history_test.py",
               "test/task_test.py",
               "test/worker_external_task_test.py",
               "test/worker_multiprocess_test.py",
               "test/worker_parallel_scheduling_test.py",
               "test/worker_test.py",
               "test/wrap_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/test_event_callbacks.py::TestEventCallbacks::test_failure",
               "test/test_event_callbacks.py::TestEventCallbacks::test_processing_time_handler_failure"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 31,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/central_planner_test.py",
               "test/rpc_test.py",
               "test/scheduler_visualisation_test.py",
               "test/server_test.py",
               "test/task_history_test.py",
               "test/task_test.py",
               "test/worker_external_task_test.py",
               "test/worker_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/central_planner_test.py::CentralPlannerTest::test_assistant_get_work_external_task"
          ]
     },
     {
          "PYTHON_V": 3.8,
          "BENCHMARK_NAME": "luigi",
          "BUG_NUMBER": 32,
          "TARGET_DIR": ".",
          "TEST_SUITE": [
               "test/instance_test.py",
               "test/decorator_test.py"
          ],
          "EXCLUDE": [
               "env",
               "test"
          ],
          "TARGET_FAILING_TESTS": [
               "test/instance_test.py::InstanceTest::test_unhashable_type"
          ]
     }
]